docs/installation.md

```diff
@@ -12,62 +12,108 @@ local:
+- Advanced Options: Register MambaForge as the system Python 3.X
+  These will make sure that MambaForge is easily accessible from most places on your computer.
 
 **On Linux**, it might be easier to do this straight from the terminal (<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>T</kbd>) with this one-liner:
 
 ```bash
-wget -nc https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && bash Miniconda3-latest-Linux-x86_64.sh -b && ~/miniconda3/bin/conda init bash
+wget -nc https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh && bash Mambaforge-Linux-x86_64.sh -b && ~/mambaforge/bin/conda init bash
 ```
 
 Restart the terminal after running this command.
 
-**On Macs**, you can run the graphical installer using the pkg file, or this terminal command:
+```{note}
+For other Linux architectures (arm64 and POWER8/9), replace the `.sh` filenames above with the correct installer name for your architecture. See the Download column in [this table](https://github.com/conda-forge/miniforge#mambaforge) for the correct filename.
+
+```
+
+**On Macs (pre-M1)**, you can run the installer using this terminal command:
 
 ```bash
-wget -nc https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh && bash Miniconda3-latest-MacOSX-x86_64.sh -b && ~/miniconda3/bin/conda init zsh
+wget -nc https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-x86_64.sh && bash Mambaforge-MacOSX-x86_64.sh -b && ~/mambaforge/bin/conda init zsh
+```
+
+**On Macs (Apple Silicon)**, use this terminal command:
+
+```bash
+wget -nc https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-arm64.sh && bash Mambaforge-MacOSX-arm64.sh -b && ~/mambaforge/bin/conda init zsh
 ```
 
 ## Installation methods
 
 ````{hint}
 Installation requires entering commands in a terminal. To open one:
 
-**Windows:** Open the *Start menu* and search for the *Anaconda Command Prompt* (if using Miniconda) or the *Command Prompt* if not.
+**Windows:** Open the *Start menu* and search for the *Miniforge Prompt* (if using Mambaforge) or the *Command Prompt* if not.
 ```{note}
 On Windows, our personal preference is to use alternative terminal apps like [Cmder](https://cmder.net) or [Windows Terminal](https://aka.ms/terminal).
 ```
 
 **Linux:** Launch a new terminal by pressing <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>T</kbd>.
 
-**Mac:** Launch a new terminal by pressing <kbd>Cmd</kbd> + <kbd>Space</kbd> and searching for *Terminal*.
+**Mac:** Launch a new terminal by pressing <kbd>Cmd</kbd> + <kbd>Space</kbd> and searching for _Terminal_.
+
 ````
 
 ### `conda` package
 
+**Windows** and **Linux**
+
+```bash
+mamba create -y -n sleap -c conda-forge -c nvidia -c sleap -c anaconda sleap=1.3.1
+```
+
+**Mac OS X** and **Apple Silicon**
+
 ```bash
-conda create -y -n sleap -c sleap -c nvidia -c conda-forge sleap=1.3.1
+mamba create -y -n sleap -c conda-forge -c anaconda -c sleap sleap=1.3.1
```

Comment:
Will this command not work on Macs if you include `-c nvidia`?

Reply:
Haven't tried it, correct me if I am wrong, but I believe we don't need any packages from nvidia for macs. Were you looking to standardize the installation command? I was aiming to minimize to just what we needed.